src/target/riscv/riscv-013.c

```diff
@@ -4526,15 +4563,16 @@ static int write_memory_bus_v1(struct target *target, const riscv_mem_access_arg
 
 	int result = sb_write_address(target, next_address, RISCV_DELAY_BASE);
 	if (result != ERROR_OK)
-		return result;
+		return MEM_ACCESS_FAILED; /* TODO: separate status code */
```

Comment:
`write_memory_bus_v1` returns an `int`, not the `mem_access_result_t` enum.

Reply:
Thanks! Addressed